src/main/java/org/cryptomator/cryptofs/health/dirid/OrphanDir.java

```diff
@@ -173,4 +165,12 @@ private CiphertextFileType determineCiphertextFileType(Path ciphertextPath) {
 			return CiphertextFileType.FILE;
 		}
 	}
+
+	private static MessageDigest getSha1MessageDigest() {
+		try {
+			return MessageDigest.getInstance("SHA1");
+		} catch (NoSuchAlgorithmException e) {
+			throw new InstantiationError("Every JVM needs to provide a SHA1 implementation.");
```

Comment:
`InstantiationError` is misleading. While I think that throwing an Error is legitimate in a broken JVM, static code analysis still prefers it if we don't touch them. Throw an `IllegalStateException` instead.

Reply:
Fixed in 06d7933